### PR TITLE
Update simulation alarm threshold criterion

### PR DESCRIPTION
**Pull request by raghava-pamula**
Before, even one failed simulation triggered the alarm because the feature is not used that much yet. Now, 3/5 periods of 5 minutes each must have error rate of 20% before the alarm is triggered.

### CODE REVIEW

File: bin/stacks/routing-api-stack.ts [hunk 1]
```diff
@@ -299,7 +299,8 @@ export class RoutingAPIStack extends cdk.Stack {
         },
       }),
       threshold: 40,
-      evaluationPeriods: 2,
+      evaluationPeriods: 3,
+      datapointsToAlarm: 2
```
Comment (ConjunctiveNormalForm) [inline bin/stacks/routing-api-stack.ts]: this makes the alarm even more sensitive than before, no?
previously it needs two consecutive periods evaluate to over threshold, now it only needs any two out of the latest three.

Comment (raghava-pamula) [reply bin/stacks/routing-api-stack.ts]: yeah you're right

Comment (raghava-pamula) [reply bin/stacks/routing-api-stack.ts]: changed to evaulationPeriods = 5 and datapointsToAlarm = 4

File: bin/stacks/routing-api-stack.ts [hunk 2]
```diff
@@ -326,7 +327,8 @@ export class RoutingAPIStack extends cdk.Stack {
         },
       }),
       threshold: 20,
-      evaluationPeriods: 2,
+      evaluationPeriods: 5,
+      datapointsToAlarm: 4,
```
Comment (ConjunctiveNormalForm) [inline bin/stacks/routing-api-stack.ts:331]: SEV3 should probably be lower? 3 out of 5?

Comment (raghava-pamula) [reply bin/stacks/routing-api-stack.ts]: changed it to 3 out of 5

**Review by ConjunctiveNormalForm — APPROVED** [review]

**Review by rileydcampbell — APPROVED** [review]

**Comment by raghava-pamula** [comment]
https://github.com/Uniswap/routing-api/commit/5b5322d11a214d67e8c50f0996abf7229e7feace
zach addressed it here

**Pull request closed without merging**